src/test/java/com/nvidia/spark/rapids/jni/DecimalUtilsTest.java

```diff
@@ -86,6 +86,22 @@ void simplePosMultiplyZeroByNegOne() {
     }
   }
 
+  @Test
+  void multiply128WithoutInterimCast() {
+    try (ColumnVector lhs =
+                 makeDec128Column("-8533444864753048107770677711.1312637916");
+         ColumnVector rhs =
+                 makeDec128Column("-12.0000000000");
+         ColumnVector expectedBasic =
+                 makeDec128Column("102401338377036577293248132533.575165");
+         ColumnVector expectedValid =
+                 ColumnVector.fromBooleans(false);
```

Comment:
I guess the code was not formated?

Reply:
It is formatted, but it could be improved by shifting things on the same line :) 

Thanks for pointing this out, it looked ugly